README.md

```diff
@@ -76,11 +76,12 @@ digital-repository/$ cap <stage> deploy
 ```
 
 ##### Digital Hub Export
-* Export repo to JSON formatted for InvenioRDM
-```
-./bin/rake repo_export
-```
-* Same as above, but with extra debugging information
+* To export Digital Hub data to JSON format use the below commands from the root project directory. It's critical to use
+		*both* `nohup` AND the trailing ampersand for the first command and similarly important to run the second command
+		with the correct job number. If you don't do all of this then the export will cut out when the terminal that started
+		it in the first place is closed for any reason.
+
 ```
-./bin/rake verbose debug repo_export
+nohup ./bin/rails r lib/scripts/repo_export_in_batches.rb > "$(date +'%Y-%m-%d-%H%M%S')_repo_export_in_batches.log" &
+disown <job number assigned in previous command>
```

Comment:
Does the user need to set the job number here? Or is it just `disown`? 

I prefer to use `screen` for these kind of tasks.

Reply:
Just using disown will default to whatever the last started job was so no, it's not necessary. I think it's worth keeping because there's a good chance these will end up getting run out of order at some point OR something else (who knows) that causes an issue. I figure if you are very specific with it less can go wrong.

Reply:
And, yes, `screen` will get the job done too from what I can tell, but I've just never really used it so I prefer this way since I know what to expect. If you want to add a section on running it with screen I think that would be good.